test/ranges.jl

```diff
@@ -689,4 +689,12 @@ let A = -1:1, B = -1.0:1.0
 
     @test ~A == [0,-1,-2]
     @test typeof(~A) == Vector{Int}
+
```

Comment:
Stole my *own* `end`... I need to stop thinking `git rebase` is smart enough that I don't have to think about what I'm doing...

Should be good now.